logging/bin/deploy_logging_open_openshift.sh

```diff
@@ -38,90 +38,87 @@ log_notice "Deploying logging components to the [$LOG_NS] namespace [$(date)]"
 
 
 ##################################
-# OpenShift Set-up               #
+# OpenShift-specific Set-up      #
 ##################################
-
-# OpenShift-specific set-up/config
 log_info "STEP 0: OpenShift Setup"
 logging/bin/deploy_openshift_prereqs.sh
 
+# On OpenShift force "full" TLS
+export LOG_KB_TLS_ENABLE=true
+
 ##################################
 # Event Router                   #
 ##################################
-
-# Collect Kubernetes events as pseudo-log messages?
-
 log_info "STEP 1: Event Router"
 logging/bin/deploy_eventrouter.sh
 
 
 ##################################
 # Open Distro for Elasticsearch  #
 ##################################
-
 log_info "STEP 2: Elasticsearch"
 logging/bin/deploy_elasticsearch_open.sh
 
 
 ##################################
 # ODFE Content (OpenShift)       #
 ##################################
-
 log_info "STEP 2a: Loading Content into Elasticsearch"
 logging/bin/deploy_elasticsearch_content_open_openshift.sh
 
 
 ##################################
 # Elasticsearch Metric Exporter  #
 ##################################
-
 log_info "STEP 3: Elasticsearch metric exporter"
 logging/bin/deploy_esexporter.sh
 
 
 ##################################
-# Create OpenShift Routes        #
+# Create OpenShift Route(s)      #
+# and display app URL(s)         #
 ##################################
-log_info "STEP 4a: Create OpenShift Route(s)"
-logging/bin/deploy_openshift_routes.sh
+log_info "STEP 4a: Create OpenShift Route(s) & Application URL(s)"
 
-##################################
-# Kibana Content                 #
-##################################
+OPENSHIFT_ROUTES_ENABLE=${OPENSHIFT_ROUTES_ENABLE:-true}
 
-# NOTE: For ODFE, Kibana is deployed as part of ES Helm chart
+if [ "$OPENSHIFT_ROUTES_ENABLE" == "true" ]; then
```

Comment:
If we're going to use a generic env name like this, then it should also be supported on the monitoring side.

Reply:
Sounds reasonable to me.  Make it so. ;-)